### PR TITLE
Adding Store Management capability

### PR DESCRIPTION
**Pull request by alegmarra**

### CODE REVIEW

**Comment by alegmarra** [comment]
@SebastianBogado remember squash before merge

File: docker-compose.yml [hunk 1]
```diff
@@ -1,54 +1,97 @@
...
   command: /bin/true
 
+content_db:
+  image: mongo:3.2.7
+  volumes_from:
+    - content_db__data_volume
+  ports:
+    - 11017:27017
+
+content_db__data_volume:
```
Comment (SebastianBogado) [inline docker-compose.yml:83]: This needs to be added to the README where the other data volumes are created

File: system/content/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "license": "MIT",
+  "version": "0.1.0",
+  "main": "./service.js",
+  "engines": {
+    "node": "*"
+  },
+  "dependencies": {
+    "bluebird": "^3.4.1",
+    "mongoose": "^4.5.3",
+    "seneca": "^2.0.1",
```
Comment (SebastianBogado) [inline system/content/package.json]: Trailing comma => invalid JSON

File: experience/web-api/index.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var app = require('express')();
+var bodyParser = require('body-parser');
```
Comment (SebastianBogado) [inline experience/web-api/index.js:4]: Missing dependency at `./package.json`

File: system/content/models/Content.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+//TODO complete this schema!
+// review if is better to have an Embedded document pattern where
+// {store: id, content:[{item}, {item}, {item}]}
+var contentSchema = mongoose.Schema({
+  name: {
+    type: String,
+    required: [true, 'How do they order this content?']
+  },
+  _store: {
+    type: String,
```
Comment (SebastianBogado) [inline system/content/models/Content.js]: Should be of type [`ObjectId`](http://mongoosejs.com/docs/api.html#schema-objectid-js)

File: system/content/service.js [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        _owner: store.owner,
+        img: "http://img1.rnkr-static.com/node_img/13/252106/C100/scottish-courage-mcewan-s-india-pale-ale-beers-photo-1.jpg",
+        description: "Bitter as fuck",
+        price: 7
+      })
+    });
+
+    Content.count().then((count) => {
+      if (!count) {
+        Content.create(stock)
```
Comment (SebastianBogado) [inline system/content/service.js]: Remember to always return the Promise, so that you don't break the chain

**Pull request merged**